Compilers.sln

```diff
@@ -64,6 +64,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CSharpSyntaxGenerator", "sr
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CompilersIOperationGenerator", "src\Tools\Source\CompilerGeneratorTools\Source\IOperationGenerator\CompilersIOperationGenerator.csproj", "{D0A79850-B32A-45E5-9FD5-D43CB345867A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CompilersIOperationGeneratorNew", "src\Tools\Source\CompilerGeneratorTools\Source\IOperationGeneratorNew\CompilersIOperationGeneratorNew.csproj", "{012612B9-0AEB-4525-9110-8064EC2E2CC6}"
```

Comment:
What's happening to the old one?

Reply:
I'm porting type by type. When I'm done, I'll kill the old one and rename this back.